frontend/src/component/events/EventLog/EventLogFilters.tsx

```diff
@@ -50,6 +57,22 @@ export const useEventLogFilters = (
                 value: env.name,
             })) ?? [];
 
+        const groupIdOptions =
```

Comment:
Build the group id options based on what is in query params, so basically it should only show the one that is selected/queryparams.